cmd/kubelet/app/options/options.go

```diff
@@ -250,6 +250,10 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 	if f.NodeStatusMaxImages < -1 {
 		return fmt.Errorf("invalid configuration: NodeStatusMaxImages (--node-status-max-images) must be -1 or greater")
 	}
+	// Obtain Kubelet Lock File
+	if f.ExitOnLockContention && f.LockFilePath == "" {
+		return fmt.Errorf("cannot exit on lock file contention: no lock file specified")
```

Comment:
Please change the error message to `invalid configuration: when ExitOnLockContention (--exit-on-lock-contention) is true, LockFilePath (--lock-file) must be non-empty`, so it is consistent with the rest of the validation errors.

Reply:
it's done, please help to see again, thank you!